src/io/flutter/run/FlutterDebugProcess.java

```diff
@@ -17,10 +17,7 @@
 import com.intellij.xdebugger.XDebugSession;
 import com.jetbrains.lang.dart.util.DartUrlResolver;
 import io.flutter.FlutterUtils;
-import io.flutter.actions.ReloadAllFlutterApps;
-import io.flutter.actions.ReloadFlutterApp;
-import io.flutter.actions.RestartAllFlutterApps;
-import io.flutter.actions.RestartFlutterApp;
+import io.flutter.actions.*;
```

Comment:
This should be rolled back. I'm pretty sure we settled on enumerating all imports.